lib/neo4j/active_node/has_n.rb

```diff
@@ -224,8 +226,12 @@ def association?(name)
         !!associations[name.to_sym]
       end
 
+      def parent_associations
+        superclass == Object ? {} : superclass.associations
+      end
+
       def associations
-        (@associations ||= {}).merge(superclass == Object ? {} : superclass.associations)
+        (@associations ||= parent_associations)
```

Comment:
I feel almost certain that if the specs are passing that we're missing a spec here.  I don't remember when I made this change, but I remember when I cached the superclass' associations that I had problems.

But since I can't think of what the problems are, probably we should go with this and we can fix whatever problems we come across ;)  We'll probably be asking people to do some testing of the 8.0 alpha anyway